Currency-React-Web-App/Controllers/HomeController.cs

```diff
@@ -0,0 +1,52 @@
+using Currency_React_Web_App.QuartzJobs;
+using Microsoft.AspNetCore.Mvc;
+using Quartz;
+
+namespace Currency_React_Web_App.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class HomeController : Controller
+{
+    private readonly IScheduler _scheduler;
+    
+    public  HomeController (IScheduler scheduler)
```

Comment:
I guess you should use rather: ISchedulerFactory and later on action call sth: 
 await scheduler.TriggerJob(new JobKey(""DatabaseRefresh"));